File: AdvancedMapAndroid/app/src/main/java/com/carto/advancedmap/model/City.java
package com.carto.advancedmap.model;

import com.carto.advancedmap.utils.BoundingBox;

/**
 * Created by aareundo on 22/09/2017.
 */

public class City {

    public String name;

    public BoundingBox bbox;

    public City(String name, BoundingBox bbox) {
        this.name = name;
        this.bbox = bbox;
    }
}

File: AdvancedMapKotlin/app/src/main/java/com/carto/advanced/kotlin/model/City.kt
package com.carto.advanced.kotlin.model

import com.carto.advanced.kotlin.utils.BoundingBox

/**
 * Created by aareundo on 13/07/2017.
 */
class City(val name: String, val bbox: BoundingBox) {

    val existsLocally
        get() = size != -1.0

    var size: Double = -1.0
}